app_check/src/BuiltInProviderWrapper.cs

```diff
@@ -63,6 +63,7 @@ internal class BuiltInProviderWrapper : IAppCheckProvider {
   }
 
   // This is called from the C++ implementation, on the Unity main thread.
+  [MonoPInvokeCallback(typeof(AppCheckUtil.CompleteBuiltInGetTokenDelegate))]
```

Comment:
Why is this line part of this change?

Reply:
Yeah, it is a fix for the iOS build. Similarly, the change to android_dependencies was a fix for the Android build.